packages/frontend/src/components/Search/FilterSearch.tsx

```diff
@@ -1,52 +1,51 @@
 import { useContext, useState } from 'react';
 import styled from '@emotion/styled';
-import { Box, Button, Checkbox, Collapse, Drawer, IconButton, Toolbar, Typography } from '@mui/material';
+import { Box, Button, Checkbox, Collapse, Drawer, IconButton, TextField, Toolbar, Typography } from '@mui/material';
 import List from '@mui/material/List';
 import ListItem from '@mui/material/ListItem';
 import ListItemText from '@mui/material/ListItemText';
 import TuneIcon from '@mui/icons-material/Tune';
 import DisabledByDefaultIcon from '@mui/icons-material/DisabledByDefault';
+import { DatePicker } from '@mui/x-date-pickers';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
 
-import { FilterSearchData, IItem, ItemTypeEnum } from './FilterSearchData';
+import { ESearchParameterName, FilterSearchData, IItem } from './FilterSearchData';
 import { ExpandLess, ExpandMore } from '@mui/icons-material';
 import { Colors } from '../../constants/Colors';
 import { SearchContext } from '../../contexts/SearchContext';
 import { ButtonWrapper } from '../../styles/ButtonWrapper';
 import { useTranslation } from 'react-i18next';
+import { EMimeType } from '../../constants/Data';
 
 type FilterSearchProps = {
   openFilter: boolean;
   toggleFilter: any;
 };
 
-const FilterSearchItem = (props: IItem) => {
+interface IFilterSearchItem extends IItem {
+  checkboxes: any;
```

Comment:
Any way to do _some_ typing here?

Reply:
I'll change those `any` to proper types after this PR, include other of your comments too